cmake/FindLibvterm.cmake

```diff
@@ -8,7 +8,17 @@ if(LIBVTERM_INCLUDE_DIR AND EXISTS "${LIBVTERM_INCLUDE_DIR}/vterm.h")
   file(STRINGS ${LIBVTERM_INCLUDE_DIR}/vterm.h VTERM_VERSION_MINOR REGEX "#define VTERM_VERSION_MINOR")
   string(REGEX MATCH "[0-9]+" VTERM_VERSION_MINOR ${VTERM_VERSION_MINOR})
 
-  set(VTERM_VERSION ${VTERM_VERSION_MAJOR}.${VTERM_VERSION_MINOR})
+  file(STRINGS ${LIBVTERM_INCLUDE_DIR}/vterm.h VTERM_VERSION_PATCH REGEX "#define VTERM_VERSION_PATCH")
+
+  # TODO(dundargoc): the following workaround is only needed to not error for
+  # versions 0.3.2 and smaller. We can remove this once we anticipate that
+  # 0.3.2- becomes uncommon enough to not needing to account for it.
```

Comment:
But we **require** 0.3.3 already (since we enable reflow by default)?

Reply:
Uh, not in the cmake script at least. I can bump the minimum version to 0.3.3 in that case.

Reply:
I bumped the minimum to 0.3.3. We still kinda need to keep this since the error message is super confusing without it.